README.md

```diff
@@ -497,10 +964,17 @@
     // declaration to the top of the scope,
     // which means our example could be rewritten as:
     function example() {
-      var declaredButNotAssigned;
+      let declaredButNotAssigned;
       console.log(declaredButNotAssigned); // => undefined
       declaredButNotAssigned = true;
     }
+
+    // using const and let
+    function example() {
+      console.log(declaredButNotAssigned); // => throws a ReferenceError
+      console.log(typeof declaredButNotAssigned); // => throws a ReferenceError
+      const declaredButNotAssigned = true;
+    }
     ```
 
   - Anonymous function expressions hoist their variable name, but not the function assignment.
```

Comment:
This isn't true for `let`. The TDZ still applies; it doesn't matter if the initial value is an function.

```javasscript
 console.log(anonymous);
```

will throw a ReferenceError.

Same with named function expressions below.

/cc @banderson